src/cli.js

```diff
@@ -86,14 +87,30 @@ commands.cd = (newDirectory) => {
 // Display contents of specified file.
 commands.cat = (filename) => {
   if (!filename) return errors.fileNotSpecified;
+  
+  const pathContainDirectory = (filename) => filename.split('/').length >= 2 || struct.hasOwnProperty(filename)
+  const directoryExist = (directory) => struct.hasOwnProperty(directory)
+  const getFilenameInfo = (filename) => {
+    let directory = filename.split('/');
+    const fileKey = directory.pop().split('.')[0];
+    return {
+      directory: directory.join(','),
+      fileKey: fileKey,
+    };
+  }
+  
+  if (pathContainDirectory(filename)) {
+    const { directory , fileKey } =  getFilenameInfo(filename);
+    const file = directoryExist(directory) ? systemData[fileKey] : errors.noSuchFileOrDirectory;
+    return file
+  }
 
-  const dir = getDirectory();
-  const fileKey = filename.split('.')[0];
-
-  if (fileKey in systemData && struct[dir].includes(fileKey)) {
+  if (!pathContainDirectory(filename)) {
```

Comment:
Small logical flow issue here - it might be the case that a user has accidentally mistyped a directory like so:

<img width="309" alt="Screen Shot 2020-05-01 at 9 04 57 AM" src="https://user-images.githubusercontent.com/2036040/80820381-971a0000-8b8b-11ea-8956-73ff3ec6dc2a.png">

This prints the contents of the skills directory because we made the erroneous assumption that anything before the period was a filename, and anything after the delimiter was a valid file extention. We should validate here that the text after the `.` is a valid filename before returning the value from the dictionary, and error if it's not

Reply:
Hi , next time I'll put more info in the pull request body.

I've made some change, it should now work as expected